great_expectations/expectations/core/expect_column_values_to_not_be_null.py

```diff
@@ -318,4 +318,7 @@ def _validate(
             unexpected_index_list=metrics.get(
                 f"{self.map_metric}.{SummarizationMetricNameSuffixes.UNEXPECTED_INDEX_LIST.value}"
             ),
+            unexpected_index_query=metrics.get(
+                f"{self.map_metric}.{SummarizationMetricNameSuffixes.UNEXPECTED_INDEX_QUERY.value}"
+            ),
```

Comment:
This was the fix needed to get the `unexpected_index_query` returned by Expectation.

Reply:
I know a few users that will be happy to see this change.